src/main/kotlin/com/exactpro/th2/infraoperator/util/KubernetesUtils.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2024-2024 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("KubernetesUtils")
+
+package com.exactpro.th2.infraoperator.util
+
+import com.exactpro.th2.infraoperator.spec.Th2CustomResource
+import com.exactpro.th2.infraoperator.spec.box.Th2Box
+import com.exactpro.th2.infraoperator.spec.corebox.Th2CoreBox
+import com.exactpro.th2.infraoperator.spec.estore.Th2Estore
+import com.exactpro.th2.infraoperator.spec.job.Th2Job
+import com.exactpro.th2.infraoperator.spec.mstore.Th2Mstore
+import io.fabric8.kubernetes.api.model.Namespace
+import io.fabric8.kubernetes.client.KubernetesClient
+import io.fabric8.kubernetes.client.KubernetesClientBuilder
+import mu.KotlinLogging
+import kotlin.streams.toList
+
+private val K_LOGGER = KotlinLogging.logger { }
+
+val CUSTOM_RESOURCE_KINDS =
+    setOf(Th2Estore::class.java, Th2Mstore::class.java, Th2CoreBox::class.java, Th2Box::class.java, Th2Job::class.java)
```

Comment:
note: What about Th2Dictionry kind? Is it accidentally missed in this set or it was excluded intentionally?

Reply:
It was removed intentionally. I added only Th2CustomResource descendants but Th2Dictionary extends base Kubernetes CustomResource class
```
public abstract class Th2CustomResource extends CustomResource<Th2Spec, StatusSpec> implements Namespaced 
```

Reply:
Then it probably would be more informative if you explicitly specify the type for the `CUSTOM_RESOURCE_KINDS` property